### PR TITLE
Feature/expose all entities in endpoints

### PR DESCRIPTION
**Pull request by vitorbaptista**

### CODE REVIEW

**Comment by vitorbaptista** [comment]
This is ready for review.

File: migrations/20160413110458_add_gender_and_has_published_results_to_trials.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+);
+
+exports.down = (knex) => (
+  knex.schema.table('trials', (table) => {
+    table.dropColumns([
+      'has_published_results',
+      'gender',
+    ]);
+  })
+);
```
Comment (roll) [inline migrations/20160413110458_add_gender_and_has_published_results_to_trials.js]: We need the same for `trialrecords` table.

**Comment by roll** [comment]
All good except adding a migration for `trialrecords` table.

**Comment by vitorbaptista** [comment]
Good catch! I've added the migration and merged it. Should be in prod in a few minutes.

**Pull request merged**